HelpSource/Classes/DrawGridX.schelp

```diff
@@ -0,0 +1,28 @@
+CLASS:: DrawGridX
+summary:: A sub-method of DrawGrid, used for setting up its horizontal size
+categories:: GUI>Accessories
+related:: Reference/plot, Classes/GridLines, Classes/Plotter, Classes/UserView, Classes/DrawGrid
```

Comment:
Does `Reference/plot` actually work when you click on it? The lowercase looks suspicious to me.

Test links before pushing, please.